arkouda/groupbyclass.py

```diff
@@ -639,8 +639,7 @@ def argmax(self, values : pdarray)\
             raise TypeError('argmax is only supported for pdarrays of dtype float64 and int64')
         return self.aggregate(values, "argmax")
     
-    def nunique(self, values : pdarray) \
-                    -> Tuple[Union[pdarray,List[Union[pdarray,Strings]]],pdarray]:
+    def nunique(self, values : pdarray) -> Tuple[Any,pdarray]:
```

Comment:
Should we apply this change to all the other reduction methods (`.sum()`, `.min()`, `.max()`, etc.)? I'm thinking they will encounter the same problem as `.nunique()` with GroupBy on Strings or Categorical.

Reply:
@reuster986 Good point and I vote yes

Reply:
I recommend a separate one so we can get this merged into master for @pierce314159 work that is dependent upon this bug fix